src/main/java/com/solr/clientwrapper/domain/port/api/DataIngectionServicePort.java

```diff
@@ -0,0 +1,6 @@
+package com.solr.clientwrapper.domain.port.api;
+
+public interface DataIngectionServicePort {
```

Comment:
Small typo "Injection". Optional change

Reply:
Yeah, I know. It's Ravikiran's code I have integrated as it was needed, so thought will discuss with him about that.